infra/resources/_modules/web_apps/send_func.tf

```
@@ -0,0 +1,58 @@
+locals {
+  send_func = {
+    app_settings = {
+      NODE_ENV                 = "production",
+      FUNCTIONS_WORKER_RUNTIME = "node",
+    }
+  }
+}
+
+module "send_func" {
+  source  = "pagopa-dx/azure-function-app/azurerm"
+  version = "~> 0.0"
```

We should be able to use the latest version
```suggestion
  version = "~> 2.0"
```